build-system/git.js

```diff
@@ -168,3 +158,23 @@ exports.gitCommitFormattedTime = function() {
 exports.gitStatusPorcelain = function() {
   return getStdout('git status --porcelain').trim();
 };
+
+/**
+ * Returns the merge base of the current branch off of master when running on
+ * a local workspace.
+ * @return {string}
+ */
+function gitMergeBaseLocalMaster() {
+  return getStdout('git merge-base master HEAD').trim();
+}
```

Comment:
I assume this was moved down purely because it's no longer an export. ✓

Reply:
Exactly